shared/components/graphs/graphs.component.js

```diff
@@ -215,8 +231,31 @@ class GraphsComponent extends Panel {
     }
   }
 
+  get category_average_byline(){
+    let graphs = this;
+    switch (graphs.current_route_name){
+      case 'Travel':
+        return graphs.t('summaries.average_travel_footprint')
+      case 'Home':
+        return graphs.t('summaries.average_home_footprint')
+      case 'Food':
+        return graphs.t('summaries.average_food_footprint')
+      case 'Shopping':
+        return graphs.t('summaries.average_shopping_footprint')
+      case 'TakeAction':
+        return graphs.t('summaries.average_action_savings')
+      default:
+        return graphs.t('summaries.average_footprint')
+    }
+  }
+
   get display_total_footprint(){
     let graphs = this;
+    if (graphs.current_route_name === 'GetStarted') {
+      graphs.state.show_total = false;
+    } else {
+      graphs.state.show_total = true;
+    }
```

Comment:
If you want to change state in react, you always have to use `component.getState` method - `state['something'] = value` won't update the DOM.

Another note, we should avoid changing state in getters. State variables should only change in React lifecycle events ([React lifecycle](https://facebook.github.io/react/docs/component-specs.html)), or UI events (`onClick`, `onChange`, etc) - or methods that those events call (you don't literally have to cram them into those methods).